### PR TITLE
Adds skeleton syllabus and text editor installation

### PR DESCRIPTION
**Pull request by dleehr**
Syllabus will need to be filled out and possibly moved around.
I added the text editor install instructions from SWC to the setup section, since we will use plaintext editors for READMEs and code

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -306,31 +333,88 @@
...
-<li> Download the Python 3 installer for Mac OSX.
-<li> Install Python 3 using all of the defaults for installation 
-</ol>
+	Regardless of how you choose to install it, please make sure you install Python version 3.x (e.g., 3.4 is fine).
+	<p>
+		<div class="row">
+			<div class="col-md-6">
+				<h4>
+					Windows
+				</h3>
```
Comment (hlapp) [inline index.html]: Did you copy&paste this from somewhere? Note that you're closing `h4` with an `h3`. (If this was copied from somewhere, it should probably be fixed there as well.)

File: index.html [hunk 2]
```diff
@@ -306,31 +333,88 @@
...
+				<ol>
+					<li> Open <a href="http://continuum.io/downloads">http://continuum.io/downloads</a> with your web browser.
+					<li> Download the Python 3 installer for Windows.
+					<li> Install Python 3 using all of the defaults for installation except make sure to check *Make Anaconda the default Python*.
+				</ol>
+			</div>
+			<div class="col-md-6">
+				<h4>
+					Mac OSX
+				</h3>
```
Comment (hlapp) [inline index.html]: See above.

**Review by hlapp — COMMENTED** [review]
Can you fix the erroneous `</h3>` tags?

File: index.html (hunk 1 above)
Comment (dleehr) [reply index.html]: No, they were `<h3>`s prior to my changes, as is **Setup**. So I demoted them to `<h4>`s but forgot to change the closing tags

**Pull request merged**